apache/zeppelin
# [ZEPPELIN-1258] Add Spark packages support to Livy interpreter

### What is this PR for?
Adding extra libraries to livy interpreter which isn't exist by default.

### What type of PR is it?
[ Improvement ]

### Todos
* [Test case ] - Task

### What is the Jira issue?
*  [ZEPPELIN-1258]

### How should this be tested?
- Create new livy interpreter or modify the default.
- Set `livy.spark.jars.packages` to list of maven coordinates of jars. The format for the coordinates should be groupId:artifactId:version. 

### Screenshots (if appropriate)
![](https://s31.postimg.org/px78l7ql7/Screenshot_from_2016_08_03_16_59_00.png)
![](https://s32.postimg.org/96ucb5rb9/Screenshot_from_2016_08_03_17_00_30.png)

### Questions:
* Does the licenses files need update? no
* Is there breaking changes for older versions? no
* Does this needs documentation? yes
